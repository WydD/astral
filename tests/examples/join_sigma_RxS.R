v
t=0
0
t=25
1