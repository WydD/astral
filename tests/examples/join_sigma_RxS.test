<?xml version="1.0" encoding="UTF-8"?>
<astral xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://astral.ligforge.imag.fr/schema/query.xsd">
<query t0="auto">
<sigma condition="id==1">
<streamer stype="Rsu">
<join>
<window>
<description wtype="L"/>
<source attributes="id,T,PHYID" id="S" sourceAttributes="id,T,PHYID" type="Stream"/>
</window>
<source attributes="v,PHYID" id="R" sourceAttributes="v,PHYID" type="Relation"/>
</join>
</streamer>
</sigma>
</query>
<sources>
<source attributes="id,T,PHYID" file="tests/examples/join_sigma_RxS.S" id="S" impl="FileStreamImpl" type="Stream"/>
<source attributes="v,PHYID" file="tests/examples/join_sigma_RxS.R" id="R" impl="FileRelationImpl" type="Relation"/>
</sources>
</astral>
