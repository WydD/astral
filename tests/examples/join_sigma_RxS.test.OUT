id,T,v
1,10,0
